das/redis_das.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2021-2022, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+
+package das
+
+import (
+	"bytes"
+	"context"
+	"encoding/base32"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/go-redis/redis/v8"
+
+	"github.com/ethereum/go-ethereum/crypto"
+
+	"github.com/offchainlabs/nitro/arbstate"
+	"github.com/offchainlabs/nitro/blsSignatures"
+)
+
+type RedisConfig struct {
+	RedisUrl   string        `koanf:"redis-url"`
+	Expiration time.Duration `koanf:"redis-expiration"`
+}
```

Comment:
How do we authenticate to redis?

Reply:
The underline network/SSL is secure. In the coordinator, we add a simple signature to data, so an attacker with redis access couldn't add messages (could still read & delete them). Makes sense to have this added level here too.